sentry-spring/src/main/java/io/sentry/spring/SentryExceptionResolver.java

```diff
@@ -20,9 +27,18 @@ public ModelAndView resolveException(HttpServletRequest request,
                                          HttpServletResponse response,
                                          Object handler,
                                          Exception ex) {
-
-        Sentry.capture(ex);
-
+        ContentCachingRequestWrapper cacheRequest = WebUtils.getNativeRequest(request, ContentCachingRequestWrapper.class);
+        if (cacheRequest == null) {
+            Sentry.capture(ex);
+            return null;
+        }
+        String requestBody =  new String(cacheRequest.getContentAsByteArray());
```

Comment:
Maybe not use the default charset of the JVM here? What about something like (ignoring error handling):
```java
        Charset cs = request.getCharacterEncoding() == null ? StandardCharsets.UTF_8 : Charset.forName(request.getCharacterEncoding());
        String content = StreamUtils.copyToString(cacheRequest.getInputStream(), cs);
```

Reply:
Using the charset from the request if available makes sense to me. But also, how do we guarantee this body can be converted to String?
Also, since this potentially allocating a huge string and affecting the server performance this should be opt-in